<launch>
    <node pkg="turtlebro_overheat_detector" type="heat_detector.py" name="heat_detector" output="screen">
        <param name="_wait_after_detection" value="10" />
        <param name="_ignore_heat_after_continue" value="10"/>
        <param name="threshold" value="40"/>
        <param name="control_topic" value="patrol_control"/>
    </node>
</launch>